cpu/ppc64_cpu_test.py

```diff
@@ -147,7 +147,11 @@ def smt_snoozedelay(self):
         Tests the smt snooze delay in ppc64_cpu command.
         """
         command1 = "ppc64_cpu --smt-snooze-delay | awk '{print $NF}'"
-        command2 = "cat /sys/bus/cpu/devices/cpu*/smt_snooze_delay | uniq"
+        tmp = "cpu*/smt_snooze_delay"
```

Comment:
can `tmp` be changed to something like `snz_delay_file`? Otherwise, looks good.

Reply:
@harish-24 agreed, will change and send a patch.

Reply:
@harish-24 Done. please check now.